src/Mvc/Mvc.ApiExplorer/test/DefaultApiDescriptionProviderTest.cs

```diff
@@ -1425,6 +1425,33 @@ public void GetApiDescription_ParameterDescription_ComplexDTO()
             Assert.Equal(typeof(decimal), price.Type);
         }
 
+        [Fact]
+        public void GetApiDescription_ParameterDescription_ComplexDTOFomQueryOnNestedProperties()
```

Comment:
Change to: `GetApiDescription_ParameterDescription_ComplexDTOFromQueryOnNestedProperties()`

Reply:
done